shell/src/main/java/alluxio/worker/AlluxioWorkerMonitor.java

```diff
@@ -43,7 +43,7 @@ public static void main(String[] args) {
 
     HealthCheckClient client = new WorkerHealthCheckClient(
         NetworkAddressUtils.getConnectAddress(NetworkAddressUtils.ServiceType.WORKER_RPC, conf),
-        () -> new ExponentialBackoffRetry(50, 100, 2), conf);
+        () -> new ExponentialBackoffRetry(50, 500, 10), conf);
```

Comment:
Could we move this supplier to a constant with a descriptive name like `TWO_MINUTE_EXP_BACKOFF`? It difficult to tell from the numbers how long the retries go for

Reply:
For the exponential ones I just get a rough estimate for the max time by doing `maxWait * numRetries`. Technically it will always be less than that, but it gives a good enough ballpark if the min/max are close to one another.

As for adding a constant, I can find a place to put one. Especially because the masters also share a similar time.

Reply:
added constants in the Master/WorkerMonitor classes